packages/apps-config/src/endpoints/testing.ts

```diff
@@ -76,20 +76,12 @@ export function createTesting (t: TFunction, firstOnly?: boolean): LinkOption[]
       }
     },
     {
-      info: 'datahighway-harbour',
-      text: t('rpc.test.datahighway-harbour', 'DataHighway Harbour', { ns: 'apps-config' }),
+      info: 'datahighway-harbour-chain-testnet',
+      text: t('rpc.test.datahighway.harbour', 'DataHighway Harbour', { ns: 'apps-config' }),
```

Comment:
from a maintenance perspective, it might be easier if this was named `datahighway-harbour` instead of `datahighway.harbour` since find/replace would be quicker. what do you think?
(same applies to `datahighway.spreehafen` and `datahighway.westlake`)